### DISCUSSION TITLE
Trouble running web version

### DISCUSSION THREAD

**Discussion by Mamun-Developer** [spacedriveapp/spacedrive#368] [General]
**No Library loaded** for `pnpm web dev` (`pnpm desktop dev` works well)

My Machine info:
OS: Ubuntu 22.04
Processor: AMD® Ryzen 7 5800h with radeon graphics × 16
Memory: 16GB
rustc 1.63.0
cargo 1.63.0
node: 17.9.1 (also tried with 17.0.0)

Steps to reproduce
pnpm I
pnpm prep
pnpm web dev

Is there anything I'm missing that causing this issue

**Comment by oscartbeaumont** [comment]
You need to run `pnpm web dev` in one terminal and then `cargo run -p server` in another. The server is what runs the Spacedrive core and then the web interface connects to it via websockets.

**Reply by oscartbeaumont** [reply]
Also, be aware the web interface isn't officially supported at this stage of development. It does not have all the same features as the desktop app working (such as adding a location) which makes it more of a tech demo and debugging tool than something you can use.

**Reply by Mamun-Developer** [reply]
thanks, it solved my issue. I think we should update the `CONTRIBUTING.md` with it.
